adoc/quickstart3_chap_suma_keys_and_first_client.adoc

```diff
@@ -46,54 +46,67 @@ endif::[]
 
 == Introduction
 
-Releases prior to {productname} 2.1 were shipped with a *traditional* client management framework (which is still fully supported today).
-SUSE Manager 3 and later include the Salt framework in addition to the *traditional* framework.
+Clients can be either *traditional* or *Salt*.
+If you are using {productname}{nbsp}2.1 or earlier, only traditional clients are supported.
+For {productname}{nbsp}3 and later, you can choose to use either traditional or Salt clients, or a mixture of both, depending on your environment and requirements.
+
 
```

Comment:
1st and 2nd sentence can be omitted in the 3.x documentation (esp. in the quickstart guide).